### DISCUSSION TITLE
No courses

### DISCUSSION THREAD

**Discussion by Skatter887** [heliomarpm/udemy-downloader-gui#102] [General]
Hello guys,

Thank you for your precious work on the app, I followed all the steps but the tab "courses" is empty, what can I do?

**Comment by heliomarpm** [comment]
> Hello guys,
> 
> Thank you for your precious work on the app, I followed all the steps but the tab "courses" is empty, what can I do?

Is your user enrolled in any courses on Udemy?

**Reply by Fadyos** [reply]
Hi,

If you read this, I'd like to firstly thank you for your time. I enrolled in a few courses with my account, but same issue than Skatter887, there's not any course appearing in Udeler. Could this be fixed ?

**Comment by Daniel-Vitanza** [comment]
Hello,

For info in my case, the soft works on 1.10.3 with all courses displayed. But nothing anymore on 1.12.3.
It is therefore not a udemy profile issue. ;)

**Reply by heliomarpm** [reply]
> Olá
> 
> Para obter informações no meu caso, o soft funciona no 1.10.3 com todos os cursos exibidos. Mas nada mais em 1.12.3. Portanto, não é um problema de perfil da udemy. ;)

Is your user a subscriber or a single purchase?
As of version 1.12.* an option was added at login to inform whether or not you are a subscriber.

try this please

**Reply by Daniel-Vitanza** [reply]
I am not a subscriber. 
But I tried with and without the option in version 1.12 to check. The course tab remains empty in both cases.

In version 1.10.3 , it is direct without issue.

**Reply by heliomarpm** [reply]
> I am not a subscriber. But I tried with and without the option in version 1.12 to check. The course tab remains empty in both cases.
> 
> In version 1.10.3 , it is direct without issue.

This is a really strange behavior, could you send me an email, so we can try together to find a solution?

**Comment by EduardoMaidana771** [comment]
Hi, I have Udemy Business and the same thing is happening to me. I can't see any courses. I've already tried versions 1.10.3 and the latest one, which is 1.12.3. Is there any solution to this? The login works perfectly on version 1.12.3, but no courses appear. On the other hand, on version 1.10.3, the login works, but it gets stuck on the loading courses part.

Additional information: In the Logger tab, I see the following error message after logging in

`uncaughtException
TypeError: Cannot read property 'checkNewVersion' of undefined at loadSettings (file:///C:/Users/User/Downloads/udemy-downloader-gui-master/assets/js/app.js:1497:91) at HTMLAnchorElement. (file:///C:/Users/User/Downloads/udemy-downloader-gui-master/assets/js/app.js:1416:3) at HTMLAnchorElement.dispatch (C:\Users\User\Downloads\udemy-downloader-gui-master\node_modules\jquery\dist\jquery.js:5494:27) at HTMLAnchorElement.elemData.handle (C:\Users\User\Downloads\udemy-downloader-gui-master\node_modules\jquery\dist\jquery.js:5298:28)`

**Reply by heliomarpm** [reply]
a new corrective version is being published

**Comment by heliomarpm** [comment]
Check the latest version

**Reply by Daniel-Vitanza** [reply]
Works perfectly now with 1.12.4. Nice work, thx a lot !

**Reply by nanna3210** [reply]
can you download course with a  subscription plan ?
